packages/services/api/src/modules/schema/providers/schema-manager.ts

```diff
@@ -969,9 +969,8 @@ export class SchemaManager {
       return null;
     }
 
-    return this.storage.getOrganizationUser({
-      organizationId: args.organizationId,
-      userId: args.userId,
+    return this.storage.getUserById({
+      id: args.userId,
```

Comment:
Method was only used here and did not provide a real benefit over the exiting one..